knowledge-base/error-in-web-site-project-failed-to-setup-service-system-invalidcastexception.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Error is thrown on attempt to add Report Viewer control on to the page.
+description: Attempting to add a Report Viewer control on the design surface of a WebForm leads to an error being thrown.
+type: troubleshooting
+page_title: Trying to add a Report Viewer control in a Web Site projects leads to an error
```

Comment:
Trying to add a Report Viewer control in a Web Site projects leads to InvalidCastException error.